src/components/ContactsList/ContactsSubList.jsx

```diff
@@ -1,17 +1,22 @@
 import React from 'react'
 import PropTypes from 'prop-types'
 
+import List from 'cozy-ui/transpiled/react/MuiCozyTheme/List'
+
 import ContactRow from './ContactRow'
 
 const ContactsSubList = ({ contacts }) => {
   return (
-    <ol className="sublist-contact">
-      {contacts.map(contact => (
-        <li key={`contact-${contact._id}`}>
-          <ContactRow id={contact._id} key={contact._id} contact={contact} />
-        </li>
+    <List>
+      {contacts.map((contact, index) => (
+        <ContactRow
```

Comment:
Nit: ContactRow -> ContactListItem. It's clearer this way that we are rendering ListItems.